movement/sample_data.py

```diff
@@ -257,11 +280,21 @@ def fetch_dataset(
     """
     file_paths = fetch_dataset_paths(filename)
 
-    ds = load_poses.from_file(
-        file_paths["poses"],
-        source_software=metadata[filename]["source_software"],
-        fps=metadata[filename]["fps"],
-    )
+    if "poses" in file_paths:
```

Comment:
Now that we've commented out the bboxes code, we could drop the `if` statement for now.